### DISCUSSION TITLE
Allow editing calendar url

### DISCUSSION THREAD

**Discussion by cyberarm** [bitfireAT/icsx5#109] [Ideas]
I subscribe to some calendars that periodically invalidate the url token, e.g. `https://example.com/forum/calendar/2-meetings/download/?key=<token>`, it'd be nice if I could just update the url instead of needing to delete and recreate the calendar to get an updated url in.

**Comment by ArnyminerZ** [comment]
Mmmmh... Such an specific use-case, but I don't see why this couldn't be done. What do you think @rfc2822 ?

**Reply by rfc2822** [reply]
Don't know about the UI and whether it could confuse people or so, but otherwise there shouldn't be a problem with editing the URL

**Reply by ArnyminerZ** [reply]
Maybe we can create a sub-screen or something, for not overloading the main edit screen. How about waiting until #91 so we can further discuss the design changes?
